lib/checks/aria/aria-valid-attr-value.json

```diff
@@ -11,7 +11,7 @@
         "plural": "Invalid ARIA attribute values: ${data.values}"
       },
       "incomplete": {
-        "noId": "ARIA attribute element ID does not exist on the page: ${data.needsReview}",
+        "noId": "ARIA attribute element ID does not exist on the page or is a descendant of a different shadow DOM tree: ${data.needsReview}",
```

Comment:
I'm not sure about this. Could we test that shadow DOM exists on the page before we include this? I don't think we should tell people that shadow DOM might be the problem if we they have no shadow DOM on the page.